README.md

```diff
@@ -1 +1,31 @@
-# kiwi-hackathon
\ No newline at end of file
+# Plantae 🌱
+**Plantae** is a web application that allows you to identify and learn about almost any plant you find.
+
+## Getting Started
+Don't know how the UI will work yet, will update this section later.
+
+## Installation
```

Comment:
probably want to add any requirements to the requirements.txt doc so that people will know what versions to download
@devyboy  and @gideontong